go-app-chw.js

```diff
@@ -436,9 +440,23 @@ go.utils = {
         return formatter();
     },
 
+    get_nurse_id: function(contact) {
```

Comment:
It's close, but the extra names differ `nc_` and there is no 'none' option for nurses.